src/NuGet.Core/NuGet.PackageManagement/IDE/PackageRestoreContext.cs

```diff
@@ -27,6 +30,19 @@ public class PackageRestoreContext
             EventHandler<PackageRestoreFailedEventArgs> packageRestoreFailedEvent,
             IEnumerable<SourceRepository> sourceRepositories,
             int maxNumberOfParallelTasks,
+            ILogger logger) : this(nuGetPackageManager, packages, token, packageRestoredEvent, packageRestoreFailedEvent, sourceRepositories, maxNumberOfParallelTasks, true, new Dictionary<string, RestoreAuditProperties>(), logger)
```

Comment:
Can this pass an empty dictionary instead of new'ing one up?

Reply:
I'll create a static.